net/bmx7/Makefile

```diff
@@ -0,0 +1,165 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bmx7
+PKG_VERSION:=7.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/bmx-routing/bmx7/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=5f88df1c95e5cb842a6016bb1604e3e7f6097c63c5c9916edc3c84e96d4f5f65
+PKG_UNPACK:=$(TAR) -C $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION) \
```

Comment:
Can be replaced with PKG_BUILD_DIR

Reply:
I tried but apparently a folder named `src/` is treated differently, I solved it via a prepare statement.

Reply:
Ah yes you're right. There was a subdirectory make flag. For CMake, this is CMAKE_SOURCE_SUBDIR. I'll look into this later

Reply:
@neheb Hey you came up with something?

Reply:
MAKE_PATH:=src is what you want I think.

Reply:
Works, thank you!